web/src/util/normalize-item.js

```diff
@@ -0,0 +1,12 @@
+export default function normalizeItem(item) {
+  return Object.assign(
```

Comment:
Out of curiosity, what's the need for `Object.assign` here?

Reply:
Ah yes, was merging objects, but decided not to. Can just be a plain object literal.

Reply:
Object doesn't throw an error on null or undefined sources, so it; just reducing the chance for thrown excpeptions.